Pocket/pocket-ios
# fix(Localization) Update offline search to 2 separate strings to aid …

…localization

## Summary
Spanish localization was difficult with the old method so we are moving from 3 strings;
1 main, 2 parameters
to 2 complete strings.

## References 
[Smartling Thread](https://dashboard.smartling.com/app/projects/3330222bb/string-details/index.html?hashcode=96cffb21ca9ab8d974ce289b500ab239&activeIssue=42b3936d5137&redirectOnCloseUrl=https%3A%2F%2Fdashboard.smartling.com%2Fapp%2Faccounts%2Fd1655600%2Fissues%2F%3FprojectIds%3D3330222bb)
https://getpocket.atlassian.net/browse/IN-1354


## Test Steps
As a Premium user navigate to Search, enable Airplane mode and search Archived and All items to see the new text.
It should appear exactly as the previous implementation did.

## PR Checklist:

## Screenshots
